assets/src/edit-story/karma/cuj/lasso.cuj.karma.js

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../';
+import { useStory } from '../../app/story';
+import { useInsertElement } from '../../components/canvas';
+import createSolid from '../../utils/createSolid';
+import useCanvas from '../../components/canvas/useCanvas';
+import {
+  dataToEditorY as dataToEditorYorg,
+  dataToEditorX as dataToEditorXorg,
+} from '../../units';
+import { PAGE_WIDTH, PAGE_HEIGHT } from '../../constants';
+
+let dataToEditorX;
+let dataToEditorY;
+
+fdescribe('CUJ: Lasso selection', () => {
+  let fixture;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+
+    await fixture.render();
+    await new Promise((r) => setTimeout(r, 0)); // to get correct pageSize, can be moved to render?
```

Comment:
Definitely. If the `pageSize` needs an extra loop, we should just move this wait to the `fixture.js`'s `render()`. Ideally we'd figure out how to wait for that event explicitly.